.github/workflows/post-release.yml

```
@@ -28,11 +28,11 @@ jobs:
           current_version=$(cargo metadata --format-version 1 --no-deps | \
             jq --raw-output '.packages | .[] | select(.name == "bevy").version')
           # Sanity check: current version should be 0.X.Y
-          if ! grep -q '^0\.[0-9]\+\.[0-9]\+$' <<< "${current_version}"; then
-            echo "Invalid version (not in 0.X.Y format): ${current_version}"
+          if ! grep -q '^0\.[0-9]\+\.[0-9]\+-dev$' <<< "${current_version}"; then
+            echo "Invalid version (not in 0.X.Y-dev format): ${current_version}"
             exit 1
           fi
-          minor_version=$(sed 's/^0\.\([0-9]\+\).*/\1/' <<< "${current_version}")
+          minor_version=$(sed 's/^0\.\([0-9]*\)\..*/\1/' <<< "${current_version}")
```

```suggestion
          minor_version=$(sed 's/^0\.\([0-9]\+\)\..*/\1/' <<< "${current_version}")
```
Could you maybe try this just once? I really don't want the sanity check to be brittle

---

I removed that change... doesn't work locally, but it does in CI